packages/app/src/users/dataAccess/dataSource.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2022 Byndyusoft
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+import { UserDto } from "ᐸDtosᐳ";
+
+export class DataSource {
```

Comment:
Can we delete them and return fake data in Command and Query? See [this commit](https://github.com/Byndyusoft/nest-template/tree/21693ed2f23d74a69ba8e38344ba2b2b409cba8e) as example.